config.js

```diff
@@ -50,6 +50,15 @@ const ConfigOptions = () => {
             stripDebug
         },
 
+        /**
+         * f-logger-related variables
+         */
+        logger: {
+            dir: 'JavaScript',
```

Comment:
What path does the `js-error.js` live at?  Is it at somethign like `/JavaScript/Shared/js-error.js`?

Only wondering as that looks like a different directory to the rest of the JS on the site?

Reply:
Caught up with Simon on this and there are a few areas that require this path to be set like this (below), I think we should raise a ticket to correct these paths and then we could update this so it lives in the `JS` folder, what do you think?

```
Ha-proxy
NXlog
Publicweb
CW
Kibana
```

Reply:
Does Publicweb hit the global logging file then?

Would be good to update anywhere that needs it too though – having JS in two places just adds confusion for people new to the project as the file will just get lost otherwise.

Reply:
Have removed this. I'll update the jira so we can update the paths for all the other utilities that used the old path.